project-base/storefront/components/Pages/Search/SearchPageContent.tsx

```diff
@@ -0,0 +1,46 @@
+import { SearchProducts } from './SearchProducts';
+import { LastVisitedProducts } from 'components/Blocks/Product/LastVisitedProducts/LastVisitedProducts';
+import { SkeletonPageSearch } from 'components/Blocks/Skeleton/SkeletonPageSearch';
+import { Webline } from 'components/Layout/Webline/Webline';
+import { SearchContent } from 'components/Pages/Search/SearchContent';
+import { useSearchQueryApi } from 'graphql/generated';
+import { useQueryParams } from 'hooks/useQueryParams';
+import useTranslation from 'next-translate/useTranslation';
+import { usePersistStore } from 'store/usePersistStore';
+
+export const SearchPageContent: FC = () => {
+    const { t } = useTranslation();
+    const { searchString } = useQueryParams();
+    const userIdentifier = usePersistStore((state) => state.userId)!;
+
+    const [{ data: searchData, fetching }] = useSearchQueryApi({
+        variables: {
+            search: searchString!,
+            isAutocomplete: false,
+            userIdentifier,
+        },
+        pause: !searchString,
+    });
```

Comment:
Would it be safer to add `userIdentifier` to pause as well, or that is not necessary?

Reply:
UserIdentifier is set during store hydration so it is always defined. I've marked `userIdentifier` as defined manually because it is always defined. With this TS is complaining about unnecessary check. I would leave it as it is.

Reply:
Alright, NP.